integrations/src/test/resources/io/jenkins/plugins/casc/SeedJobTest_withSecurityConfig.yml

```diff
@@ -0,0 +1,26 @@
+security:
+  GlobalJobDslSecurityConfiguration:
+    useScriptSecurity: false
+
+jobs:
+  - file: ./src/test/resources/io/jenkins/plugins/casc/testJob2.groovy
+  - script: >
+      freeStyleJob('seedJobWithSecurityConfig') {
+        steps {
+            customWorkspace("$SEED_JOB_FOLDER_FILE_PATH")
+            dsl(['testJob2.groovy'], 'DISABLE')
+        }
+      }
+  - script: queue("seedJobWithSecurityConfig")
+
+# metrics plugin blocks build otherwise if left out
```

Comment:
@batmat :see_no_evil:

Reply:
umm...? 
Is that a transitive dependency of the newer job dsl plugin?

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/4fd1a6990eaf4c920f714dc5ee795d78eef483cf/integrations/pom.xml#L47-L52
Metrics plugin ended up having a single integration test in our repo. 😢 Which affects build from getting assertion errors in builds and fails our integration test builds 😰

Reply:
o/  Not sure what is expected from me? Was I the one putting this comment? I do not remember TBH. Thanks

Reply:
@batmat I added the comment since I found it odd that I would hit AssertionError: https://github.com/jenkinsci/metrics-plugin/blob/b69110b9b8115544a5b02b9709ca15df6ceab3bb/src/main/java/jenkins/metrics/api/Metrics.java#L188-L195

Running builds just because the metrics plugin installed as test dependency!

Reply:
Not sure what triggers this, not being a `metrics` plugin maintainer. I would assume something statically loads the plugin and hence fail if unconfigured or so. 

Possibly @dwnusbaum would know more about this. And if this is expected or a potential bug.

Reply:
I would be keen on removing the metrics plugin test here and move it to the plugin instead 😄

Reply:
I must have seen :ghost: